### DISCUSSION TITLE
Is zlib-ng based on the latest official zlib?

### DISCUSSION THREAD

**Discussion by Spatzendame** [zlib-ng/zlib-ng#963] [General]
The zlib-ng version number is 1.2.11 which seems to be based on the latest official zlib release. But the copyright says 1995-2013, while normally it should be 1995-2017.
So zlib-ng may based on zlib 1.2.8 only?

Can you explain this?
Thank you!

**Comment by mtl1979** [comment]
zlib-ng was forked from zlib before 1.2.11 was released, but we have integrated relevant changes that didn't conflict with any zlib-ng specific changes already made since the fork happened.

**Reply by Spatzendame** [reply]
Okay, thank you for the information. So zlib-ng is the definitive version with the most relevant fixes?
What is the zlib1.dll version that zlib-ng is originally based on?

**Reply by mtl1979** [reply]
I don't think there is any specific dll version for the commit when zlib-ng was forked... Last common commit I could find in the repositories was dated 15th September 2014.
